_includes/integrations/react-tech-ref.md

```diff
@@ -150,22 +217,45 @@ For information on the {{site.productname}} selector (`tinymce.init`), see: [Bas
 ```
 
 #### `initialValue`
-Initial content of the editor when the editor is initialized.
 
-**Default value:** `' '`
+The initial HTML content of the editor. This will reset the editor undo state and cursor when changed.
+
+This may be set either before the editor loads, or soon afterwards by an asynchronous
+process.
+
+Ensure that this is **not** updated by the editor value or the editor will be unusable.
```

Comment:
Should this be an admonition? ie be prefixed with `> **Important**:`

Reply:
done